Formula/bigreqsproto.rb

```diff
@@ -17,7 +17,7 @@ class Bigreqsproto < Formula
 
   # Patch for xmlto
   patch do
-    url "https://raw.githubusercontent.com/Linuxbrew/homebrew-xorg/master/patch_configure.diff"
+    url "https://raw.githubusercontent.com/Linuxbrew/homebrew-xorg/master/Formula/patch_configure.diff"
```

Comment:
Patches could go in their own subdirectory. I usually prefer to `gist` them myself.

Reply:
I like the idea of `Patches` folder :)
Gists are fine too though I think keeping patches affiliated with the project is a better approach.

Reply:
Either is fine by me. Be sure to link to an absolute revision of the patch and not `master`.

Reply:
okeydoke

Reply:
I guess the patch has be to moved to Patches directory first to record the commit number

Reply:
Good point. That may be why upstream uses a separate repo to store patches.